packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -300,9 +323,78 @@ const VirtualListBaseFactory = (type) => {
 			return -1;
 		}
 
+		findSpottableItemWithPositionInExtent = (indexFrom, indexTo, position) => {
+			const
+				{dataSize} = this.props,
+				{dimensionToExtent} = this.uiRef;
+
+			if (0 <= indexFrom && indexFrom < dataSize &&
+				-1 <= indexTo && indexTo <= dataSize &&
+				0 <= position && position < dimensionToExtent) {
+				const
+					{isItemDisabled} = this.props,
+					direction = (indexFrom < indexTo) ? 1 : -1,
+					delta = direction * dimensionToExtent,
+					diffPosition = (indexFrom % dimensionToExtent) - position,
+					// When direction is 1 (forward) and diffPosition is positive, add dimensionToExtent.
+					// When direction is -1 (backward) and diffPosition is negative, substract dimensionToExtent.
+					startIndex = indexFrom - diffPosition + ((direction * diffPosition > 0) ? (direction * dimensionToExtent) : 0);
```

Comment:
(direction * dimensionToExtent) -> Isn't this delta?

Reply:
YES! 😅